R/last.R

```diff
@@ -1,8 +1,22 @@
 # data.table defined last(x) with no arguments, just for last. If you need the last 10 then use tail(x,10).
 # for xts class objects it will dispatch to xts::last
 # reworked to avoid loading xts namespace (#3857) then again to fix dispatching of xts class (#4053)
-last = function(x, n=1L, ...) {
+last = function(x, n=1L, na.rm=FALSE, ...) {
   verbose = isTRUE(getOption("datatable.verbose", FALSE))
+
+  stopifnot(isTRUEorFALSE(na.rm))
+
+  if (na.rm) {
+    if (verbose) 
+      cat("last: na.rm=TRUE\n")
+    
+    if (is.vector(x) && length(x)) {
+      x = x[complete.cases(x)]
```

Comment:
why not `x[!is.na(x)]`

Reply:
Already used `complete.cases` for the `data.frame` case. However, seeing `complete.cases` failing for non standard types likes `nanotime` `!is.na` seems more reasonable.